### PR TITLE
play 2.7 and scala 2.13 support

### PR DESCRIPTION
**Pull request by jtjeferreira**

### CODE REVIEW

**Comment by sief** [comment]
Thanks @jtjeferreira, looks good. Would you mind migrating the sample apps, too?

File: module/app/com/digitaltangible/tokenbucket/TokenBucketGroup.scala [hunk 1]
```diff
@@ -46,12 +46,12 @@ class TokenBucketGroup(size: Long, rate: Double, clock: Clock = CurrentTimeClock
...
    * Refills all buckets at the given rate. Full buckets are removed.
    */
-  private[this] def refillAll() {
+  private[this] def refillAll(): Unit = {
     val now: Long = clock.now
     val diff: Long = now - lastRefill
     val tokensToAdd: Long = (diff * ratePerNano).toLong
     if (tokensToAdd > 0) {
-      buckets = buckets.mapValues(_ + tokensToAdd).filterNot(_._2 >= size)
+      buckets = buckets.mapValues(_ + tokensToAdd).filterNot(_._2 >= size).toMap
```
Comment (sief) [inline module/app/com/digitaltangible/tokenbucket/TokenBucketGroup.scala:54]: can you explain why you inserted `toMap` here?

Comment (jtjeferreira) [reply module/app/com/digitaltangible/tokenbucket/TokenBucketGroup.scala:54]: because `mapValues` in 2.13 returns a `MapView`

**Comment by jtjeferreira** [comment]
> Thanks @jtjeferreira, looks good. Would you mind migrating the sample apps, too?

Done

**Comment by sief** [comment]
thanks again @jtjeferreira, I'll try to make a new release tomorrow

**Pull request merged**